### ISSUE TITLE
Warning when import angr

### ISSUE THREAD

**Issue by ramhillel** [angr/angr#697]
I get this message when trying to import angr from python interpreter:
Python 2.7.14 (v2.7.14:84471935ed, Sep 16 2017, 20:25:58) [MSC v.1500 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import pyvex
>>> import capstone
>>> import unicorn
>>> import angr
WARNING | 2017-10-09 15:12:25,463 | [32mangr.state_plugins.unicorn_engine[0m | [32mfailed loading "angr_native.dll", unicorn support disabled ([Error 126] The specified module could not be found: [Error 126] The specified module could not be found)[0m

all other dependencies loaded without errors.
I'm using python 64-bit on windows 64-bit.
Thanks

**Comment by ltfish** [comment]
I think it failed to build `angr_native.dll`, which is under `angr\native`. It should be built by calling `nmake` on `Makefile-win`. You can try to build it again by manually invoking the `_build_native()` method in angr's `setup.py`.

**Comment by ramhillel** [comment]
you're right. installed it from local instead from pip install and it worked.

**Comment by ltfish** [comment]
I think it's still an issue that we want to fix with the PyPI install. Re-opening this issue to track the fix (if any of us is not busy and wants to give this a shot).
